menu/scene_tabs.go

```diff
@@ -61,8 +61,8 @@ func buildTabs() Scene {
 		subLabel: "Scan your collection",
 		icon:     "add",
 		callbackOK: func() {
-			usr, _ := user.Current()
-			menu.Push(buildExplorer(usr.HomeDir, nil,
+			home, _ := os.UserHomeDir()
+			menu.Push(buildExplorer(home, nil,
```

Comment:
@RobLoach I think we should leave this one unchanged? When selecting directory to scan it would lists from /home/user instead of snaps/flatpaks $HOME and it wouldn't affect other systems.

Reply:
Interesting, had no clue Golang messed with that. Surprised `user.Current().HomeDir != os.UserHomeDir`

:upside_down_face: